src/common/AndroidEnvironmentRequirements.ts

```diff
@@ -16,27 +17,76 @@ const messages = Messages.loadMessages(
     'requirement-android'
 );
 
-export class AndroidEnvironmentRequirements implements RequirementList {
+export class AndroidSDKRootSetRequirements implements RequirementList {
     public requirements: Requirement[] = [];
     public enabled = true;
-    constructor(logger: Logger, apiLevel?: string) {
+    public title = 'Android SDK Requirements';
+    constructor(logger: Logger) {
         this.requirements = [
             new AndroidSDKRootSetRequirement(logger),
-            new Java8AvailableRequirement(logger),
-            new AndroidSDKToolsInstalledRequirement(logger),
-            new AndroidSDKPlatformToolsInstalledRequirement(logger),
-            new PlatformAPIPackageRequirement(logger, apiLevel),
-            new EmulatorImagesRequirement(logger, apiLevel)
+            new AndroidSDKInstallTask(logger)
+        ];
+    }
+}
+
+export class Java8AvailableRequirements implements RequirementList {
+    public requirements: Requirement[] = [];
+    public enabled = true;
+    public title = 'Java 8 Requirements';
```

Comment:
If these strings are user facing then they should be coming from the message file

Reply:
I could consider adding that to this PR.  But thought I already let scope creep enough.  I can add a ticket for followup afterwards.

Reply:
Adding them to the message file is really small work. You can just copy-paste them there and then pull them here. It would be nice to include that as part of this PR for a complete work.